### PR TITLE
fix: add devpatricia

### PR DESCRIPTION
**Pull request by devPatricia**
resolve #issueNumber #1168

resolves #issueNumber

## Checklist for profile addition

- [x] Name the PR `fix: add <your-username>`.
- [x] You've added file named `your-username.json` in the [profiles](https://github.com/EddieHubCommunity/awesome-github-profiles/tree/main/profiles) directory.
- [x] You've linked the PR to the correct issue.

---

## Other additions/changes

- [x] Changes made in this PR are mentioned below
- [x] Search previous suggestions before making a new one, as yours may be a duplicate.
- [x] I acknowledge that all my contributions will be made under the project's license.
- [x] I've read the [contributing guidelines](https://github.com/EddieHubCommunity/awesome-github-profiles/blob/main/CONTRIBUTING.md)

### CODE REVIEW

File: profiles/patriciadiaz.json [hunk 1]
```diff
@@ -0,0 +1,7 @@
+{
+    "image": "https://user-images.githubusercontent.com/15069545/173307507-7917cca8-4dba-465b-bb3c-321d82cb5997.png",
+    "issueId": 1168,
+    "name": "Patricia",
+    "username": "devpatricia"
+  }
+
```
Comment (Pradumnasaraf) [inline profiles/patriciadiaz.json:7]:
```suggestion
}
```

**Review by Pradumnasaraf — CHANGES_REQUESTED** [review]
Hey! Everything looks good, but some minor changes needed :smile: 

__
[sema-logo](https://semasoftware.com/gh) &nbsp;**Summary:** :ok_hand: This code looks good

**Comment by pranay202** [comment]
Hey @devPatricia, everything looks good. Just add one extra line at the end of the code.

File: profiles/patriciadiaz.json [hunk 2]
```diff
@@ -0,0 +1,6 @@
+{
+    "image": "https://user-images.githubusercontent.com/15069545/173307507-7917cca8-4dba-465b-bb3c-321d82cb5997.png",
+    "issueId": 1168,
+    "name": "Patricia",
+    "username": "devpatricia"
+}
```
Comment (Pradumnasaraf) [inline profiles/patriciadiaz.json:6]:
```suggestion
}

```

**Review by Pradumnasaraf — CHANGES_REQUESTED** [review]
Everything looks good to me, just add one extra line!

File: profiles/patriciadiaz.json [hunk 3]
```diff
@@ -0,0 +1,6 @@
+{
+    "image": "https://user-images.githubusercontent.com/15069545/173307507-7917cca8-4dba-465b-bb3c-321d82cb5997.png",
+    "issueId": 1168,
+    "name": "Patricia",
+    "username": "devpatricia"
```
Comment (Panquesito7) [inline profiles/patriciadiaz.json:5]: The indentation seems to be wrong here.
```suggestion
  "image": "https://user-images.githubusercontent.com/15069545/173307507-7917cca8-4dba-465b-bb3c-321d82cb5997.png",
  "issueId": 1168,
  "name": "Patricia",
  "username": "devpatricia"
```

**Review by Panquesito7 — CHANGES_REQUESTED** [review]

**Review by Panquesito7 — APPROVED** [review]

**Review by schmelto — APPROVED** [review]
## Looking good to me

Your suggested changes looking good to me.
__
[sema-logo](https://semasoftware.com/gh) &nbsp;**Summary:** :ok_hand: This code looks good&nbsp; | &nbsp;**Tags:** Elegant

**Pull request merged**